src/mnode/mnode-pastel.cpp

```diff
@@ -792,368 +857,395 @@ std::vector<CArtActivateTicket> CArtActivateTicket::FindAllTicketByArtistHeight(
 }
 
-// Art Trade Tickets ///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-// CArtSellTicket ///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-CArtSellTicket CArtSellTicket::Create(std::string _artTnxId, int _askedPrice, int _validAfter, int _validBefore, int _copy_number, std::string _pastelID, const SecureString& strKeyPass)
-{
-    CArtSellTicket ticket(std::move(_pastelID));
-    
-    ticket.artTnxId = std::move(_artTnxId);
-    ticket.askedPrice = _askedPrice;
-    ticket.activeBefore = _validBefore;
-    ticket.activeAfter = _validAfter;
-    
-    ticket.GenerateTimestamp();
+// Art Trade Tickets ///////////////////////////////////////////////////////////////////////////////////////////////////
+
+// CArtSellTicket //////////////////////////////////////////////////////////////////////////////////////////////////////
+CArtSellTicket CArtSellTicket::Create(
+    std::string _artTnxId, int _askedPrice, int _validAfter, int _validBefore, int _copyNumber,
+    std::string _recipientPastelID, std::string _pastelID, const SecureString& strKeyPass) {
+  CArtSellTicket ticket(std::move(_pastelID), std::move(_recipientPastelID));
     
-    //NOTE: Sell ticket for Trade ticket will always has copyNumber = 1
-    ticket.copyNumber = _copy_number > 0 ? 
-        _copy_number : static_cast<decltype(ticket.copyNumber)>(CArtSellTicket::FindAllTicketByArtTnxID(ticket.artTnxId).size()) + 1;
-    ticket.key = ticket.artTnxId + ":" + to_string(ticket.copyNumber);
+  ticket.artTnxId = std::move(_artTnxId);
+  ticket.askedPrice = _askedPrice;
+  ticket.activeBefore = _validBefore;
+  ticket.activeAfter = _validAfter;
     
-    std::string strTicket = ticket.ToStr();
-    ticket.signature = CPastelID::Sign(reinterpret_cast<const unsigned char*>(strTicket.c_str()), strTicket.size(), ticket.pastelID, strKeyPass);
+  ticket.GenerateTimestamp();
     
-    return ticket;
+  // NOTE: Sell ticket for Trade ticket will always has copyNumber = 1
+  ticket.copyNumber = _copyNumber > 0 ?
+    _copyNumber : static_cast<decltype(ticket.copyNumber)>(CArtSellTicket::FindAllTicketByArtTnxID(ticket.artTnxId).size()) + 1;
+  ticket.keyTwo = ticket.artTnxId + ":" + std::to_string(ticket.copyNumber);
+
+  const std::string strTicket = ticket.ToStr();
+  ticket.signature = CPastelID::Sign(
+    reinterpret_cast<const unsigned char*>(strTicket.c_str()), strTicket.size(), ticket.pastelID, strKeyPass
+  );
+
+  return ticket;
 }
 
-std::string CArtSellTicket::ToStr() const noexcept
-{
-    std::stringstream ss;
-    ss << pastelID;
-    ss << artTnxId;
-    ss << askedPrice;
-    ss << copyNumber;
-    ss << activeBefore;
-    ss << activeAfter;
-    ss << m_nTimestamp;
-    return ss.str();
+std::string CArtSellTicket::ToStr() const noexcept {
```

Comment:
Lots of the changes is about code-formatting differences only.
This makes the changes hard to review.
Does the coding convention changed intentionally?